.vscode/tasks.json

```diff
@@ -1,6 +1,65 @@
 {
     "version": "2.0.0",
     "tasks": [
+        {
+            "label": "buildIdentityApi",
+            "dependsOrder": "sequence",
+            "dependsOn": [
+                "buildIdentity",
+                "buildAPI"
+            ],
+            "problemMatcher": [
+                "$msCompile"
+            ]
+        },
+        {
+            "label": "buildIdentityApiAdmin",
+            "dependsOrder": "sequence",
+            "dependsOn": [
+                "buildIdentity",
+                "buildAPI",
+                "buildAdmin"
+            ],
+            "problemMatcher": [
+                "$msCompile"
+            ]
+        },
+        {
+            "label": "buildFullServer",
```

Comment:
❓ I may not be understanding the problem fully, but could it be valuable to always build the whole solution and then run components? What downstream complications might happen with partial builds?

Reply:
The same complications as running partial servers. There's no point in building a project that you don't run. Dependencies of projects will still be built, it's just that, for example, SSO is left out of the build when you don't start it up.